Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -159,12 +159,14 @@ end MissingEnd
 The `cdot` linter is a syntax-linter that flags uses of the "cdot" `·` that are achieved
 by typing a character different from `·`.
 For instance, a "plain" dot `.` is allowed syntax, but is flagged by the linter.
+It also flags "isolated cdots", i.e. when the `⬝` is on its own line.
```

Comment:
```suggestion
It also flags "isolated cdots", i.e. when the `·` is on its own line.
```
I believe you used `⬝` (`\cdot`) instead of  `·` (`\.` or `\centerdot`). Maybe the `"cdot"` above is confusing and should be adjusted...